### PR TITLE
Desktop page

### PR DESCRIPTION
**Pull request by LethaboMot**
In this project I:

-Added more images
- edit and added to HTML
- edit and added on CSS

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -116,3 +127,298 @@ header {
   text-decoration: none;
   font-size: 16px;
 }
+
```
Comment (christianbiring1) [inline style.css:130]: - [ ] Kindly consider using the media queries to style your page on desktop screens.
`Hint: If you do not have any clue how to do that. It's totally fine. You can check that` [here](https://www.youtube.com/watch?v=yU7jJ3NbPdA)`

Your design :point_down: 
image

Figma template :point_down: 
image

File: style.css [hunk 2]
```diff
@@ -116,3 +127,298 @@ header {
...
+.work_statement {
+  grid-column: 1/-1;
+  justify-self: center;
+  align-self: center;
+}
+
+.work_statement img {
+  max-width: 100%;
+}
```
Comment (christianbiring1) [inline style.css:161]: - [ ] Kindly make sure you stick to the design as much as possible, the spacing(margin, padding), typos are supposed to be similar to the design in Figma.
Your design :point_down: 
image

Figma template :point_down: 
image

File: style.css [hunk 3]
```diff
@@ -116,3 +127,298 @@ header {
...
+
+.details {
```
Comment (christianbiring1) [inline style.css:163]: - [ ] Kindly make sure you stick to the design as much as possible.

Your design :point_down: 
image
image

Figma design :point_down: 
image
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @LethaboMot , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] Descriptive PR `title` and `summary`
- [ ] All linters checks are passing :heavy_check_mark: 
- [ ] Good commit history.

## Required Changes ♻️
- [ ] `KINDLY CONSIDER USING THE media-query TO STYLE HOW YOUR ELEMENTS WILL LOOK LIKE ON THE DESKTOP VERSION. IF YOU ARE NOT FAMILIAR WITH IT, IT'S TOTALLY FINE. YOU MAY CONSIDER CHECKING IT` [HERE](https://www.youtube.com/watch?v=yU7jJ3NbPdA&t=41s)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Fixing your mobile version after you have applied media-query as you can see how it looks ugly 
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -7,19 +7,263 @@
...
+                </fieldset>
+            </form>
+        </section>
+
+    </main>
+
+</body>
+
+</html>
     <section id="Introduction">
```
Comment (Nemwel-Boniface) [inline index.html:267]: - [ ] I am not sure if this section added is supposed to be in your project. Is this an extension of the mobile version? it is supposed to be one markup but the styling is what will make it both mobile and desktop friendly.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @LethaboMot      :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job addressing some of the changes that were requested by a previous reviewer :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] As requested by the previous reviewer, kindly ensure that you make use of media queries to ensure that you have both a desktop and a mobile version. **Note:** even though you were supposed to build the desktop version of your project, you are required to built on top of the mobile version by adding the media queries to now help you have the desktop version together with the media query.
- [ ] When you add the media query and have the mobile version as well in your project, your project will be ready for approval., 

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -1,118 +1,504 @@
...
+  }
+
+  .headline {
+    min-height: 916px;
+    background-repeat: no-repeat;
+    display: flex;
+    justify-content: center;
+    align-items: flex;
+    margin-left: 100px;
+  }
```
Comment (jerryowusu) [inline style.css:440]: [OPTIONAL] Nice work implementing the media query. Unfortunately some of your images are not working as expected as shown  below. Kindly check and fix them.

Screenshot from 2022-11-12 03-28-36

Screenshot from 2022-11-12 03-28-57

Screenshot from 2022-11-12 03-29-13

**Review by jerryowusu — APPROVED** [review]
Hi @LethaboMot 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

power approve


## Highlights
All linters are passing :heavy_check_mark: 
Good commit messages :heavy_check_mark: 
Followed GitHub flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**